src/interactive.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable import/prefer-default-export */
+
```

Comment:
- [ ] As the previous reviewer suggested kindly make sure to remove all the `eslint-disable` rules from your code so all the linters are passing.
  - [ ] The first one is caused because you're importing the `tasks` and `todoList` from `script.js`, and `script.js` imports 'interactive.js`. (import cycle)
  - [ ] You can do `export default function displayTasks() {...` to fix the second error